lib/bundlex/project.ex

```diff
@@ -21,8 +31,15 @@ defmodule Bundlex.Project do
   * `includes` - Paths to look for header files (empty list by default).
   * `lib_dirs` - Paths to look for libraries (empty list by default).
   * `libs` - Names of libraries to link (empty list by default).
+  * `os_deps` - List of external OS dependencies. It's a keyword list, when each key is the
```

Comment:
```suggestion
  * `os_deps` - List of external OS dependencies. It's a keyword list, where each key is the
```
(alternatively we can change `when` into `which`